modules/documents/client/directives/documents.manager.directive.js

```diff
@@ -228,7 +229,15 @@ angular.module('documents')
 					self.selectNode(doc.model.id);
 				};
 
-				self.selectNode = function (nodeId) {
+				self.gotoDoc = function(docID) {
+					Document.lookup(docID) //docID is an objectID
```

Comment:
I believe the search results only return results that the user can read. Is this correct assumption?